docs/pages/guides/using-next-cloudinary-with-custom-domains.mdx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import Head from 'next/head';
+import { CldImage } from 'next-cloudinary';
+import OgImage from '../../components/OgImage';
+import styles from '../../styles/Docs.module.scss';
+
+<Head>
+  <title>Using Next-Cloudinary with Custom Domains</title>
+  <meta name="og:title" content="Using Next-Cloudinary with Custom Domain" />
+  <meta name="og:url" content={`https://next.cloudinary.dev/guides/using-next-cloudinary-with-custom-domains`} />
+</Head>
+
+<OgImage
+  title="Using Custom Domains - Next Cloudinary"
+  twitterTitle="Using Next-Cloudinary with Custom Domains"
+/>
+
+# Using Next-Cloudinary with Custom Domains
+
+When using Next-Cloudinary with a custom domain in your Next.js application, you can serve your media assets with your own branded URLs. 
+
+**Note: This feature is relevant only for users on Advanced plans or higher**
+
+## Example
+
+To display images and videos using your custom domain with the `CldImage` or `CldVideoPlayer` component.
+
+### Setup Your Custom Domain 
+
+1. **Configure Your Custom Domain**: Set the Custom Domain in the Cloudinary dashboard.
```

Comment:
you actually have to reach out to support for this. can you add something along the lines of "Request a custom domain with Cloudinary. Learn more" and link to this https://support.cloudinary.com/hc/en-us/articles/202520562-Can-we-deliver-our-assets-using-a-subdomain-of-our-own-domain-like-cdn-example-com

Reply:
Added